### DISCUSSION TITLE
show budget in transaction list

### DISCUSSION THREAD

**Discussion by albaloo2000** [firefly-iii/firefly-iii#6338] [Firefly III]
Hello, 

I have another questions.  I would like to see the budget each transaction belongs to in the list format below.  I looked at the options but budget is not listed one of them. 

<img width="1220" alt="image" src="https://user-images.githubusercontent.com/16837681/184202065-3d5f05c5-fba7-43a9-b492-b54fe3f25751.png">

Looks like the option to show the budges is in the code:

        {% if showCategory %}
            <th class="hidden-xs">{{ trans('list.category') }}</th>
        {% endif %}
        {% if showBudget %}
            <th class="hidden-xs">{{ trans('list.budget') }}</th>
        {% endif %}

But I don't see where in the program to enable this.  Also looked around in the database and I don't see where this option is stored to manually enable. 

Thanks in advance.

**Comment by JC5** [comment]
It's not possible. It will be in the new layout.

**Comment by albaloo2000** [comment]
Thank you for the quick reply.  I can't imagine how many hours you have put into this program.

**Reply by JC5** [reply]
Way too many 😅

**Comment by albaloo2000** [comment]
I set showBudget to true and it is indeed showing me the budgets.  I assume you had some kind of error that you disabled it.

I also see you have a bunch of vue components but the page shows no vue loaded.  Is the new layout an SPA using vue?

**Reply by JC5** [reply]
Yes, but the create and update a transaction form are also Vue components. The profile page also has some.
